### PR TITLE
Add tests for useStoredState

### PR DESCRIPTION
**Pull request by manuelblum**
## What

Adds Vitest tests for `useStoredState` — a localStorage-backed state hook exported from `@comet/admin`.

## Tests added

`packages/admin/admin/src/hooks/__tests__/useStoredState.test.ts` (9 tests):

- **Happy path**: returns initial value when nothing is stored; returns stored value when key exists in storage; persists state updates back to storage
- **Edge cases**:
  - `key === false` skips all storage reads/writes (in-memory only mode)
  - Malformed JSON in storage falls back to the initial value and logs the error
  - Initializer function is called when no stored value exists, but skipped when a stored value is present
  - Stored boolean `false` (persisted as the string `"false"`) is correctly restored
  - Complex object values round-trip through JSON correctly

---
_Generated by [Claude Code](https://claude.ai/code/session_01JchRRniFQXMhHeuffeoTWz)_

### CODE REVIEW

File: packages/admin/admin/src/hooks/__tests__/useStoredState.test.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        },
+        key: (index: number) => Object.keys(store)[index] ?? null,
+    };
+}
+
+afterEach(() => {
+    localStorage.clear();
+    vi.restoreAllMocks();
+});
+
```
Comment (johnnyomair) [inline packages/admin/admin/src/hooks/__tests__/useStoredState.test.ts]: I believe we can remove this since we're using a fake storage and no mocks:

```suggestion
```

**Review by johnnyomair — APPROVED** [review]
